opencl-aot/source/main.cpp

```diff
@@ -253,17 +265,46 @@ getCompilerBuildLog(const CLProgramUPtr &ProgramUPtr, cl_device_id Device) {
 
 int main(int Argc, char *Argv[]) {
   // step 0: set command line options
-  cl::opt<std::string> OptInputBinary(
-      cl::Positional, cl::Required,
-      cl::desc("<input SPIR-V or OpenCL program binary>"),
+  cl::opt<std::string> OptInputFile(
+      cl::Positional,
+      cl::desc("<input SPIR-V or OpenCL program binary or OpenCL C source>"),
       cl::value_desc("filename"));
+  cl::opt<std::string> OptOutputElf(
+      "o", cl::init("aot-output.bin"),
+      cl::desc("Specify the output OpenCL program binary filename"),
+      cl::value_desc("filename"));
+
   cl::opt<DeviceType> OptDevice(
       "device", cl::Required, cl::desc("Set target device type:"),
       cl::values(
           clEnumVal(cpu, "Intel(R) processor device"),
           clEnumVal(gpu, "Intel(R) Processor Graphics device"),
           clEnumVal(fpga_fast_emu,
                     "Intel(R) FPGA Emulation Platform for OpenCL(TM) device")));
+
+  enum Commands { build, compile, link };
+  cl::opt<Commands> OptCommand(
+      "cmd", cl::desc("Command"),
+      cl::values(
+          clEnumVal(
+              build,
+              "Build executable IR from OpenCL source or OpenCL .obj file"),
+          clEnumVal(compile, "Compile from OpenCL source to .obj"),
```

Comment:
```suggestion
          clEnumVal(compile, "Compile OpenCL C source to OpenCL program object"),
```
Please re-check it if it's true